src/routes/recovery/recovery.controller.spec.ts

```diff
@@ -0,0 +1,138 @@
+import * as request from 'supertest';
+import { faker } from '@faker-js/faker';
+import { INestApplication } from '@nestjs/common';
+import { TestingModule, Test } from '@nestjs/testing';
+import { TestAppProvider } from '@/__tests__/test-app.provider';
+import { AppModule, configurationModule } from '@/app.module';
+import { IConfigurationService } from '@/config/configuration.service.interface';
+import { TestCacheModule } from '@/datasources/cache/__tests__/test.cache.module';
+import { CacheModule } from '@/datasources/cache/cache.module';
+import { TestNetworkModule } from '@/datasources/network/__tests__/test.network.module';
+import { NetworkModule } from '@/datasources/network/network.module';
+import { NetworkService } from '@/datasources/network/network.service.interface';
+import { TestLoggingModule } from '@/logging/__tests__/test.logging.module';
+import { RequestScopedLoggingModule } from '@/logging/logging.module';
+import { addRecoveryModuleDtoBuilder } from '@/routes/recovery/entities/__tests__/add-recovery-module.dto.builder';
+import { omit } from 'lodash';
+import configuration from '@/config/entities/__tests__/configuration';
+import { AlertsModule } from '@/routes/alerts/alerts.module';
+import { RecoveryModule } from '@/routes/recovery/recovery.module';
+import { ConfigurationModule } from '@/config/configuration.module';
+
+describe('Recovery (Unit)', () => {
+  let app: INestApplication;
+  let alertsUrl: string;
+  let alertsAccount: string;
+  let alertsProject: string;
+  let networkService;
+
+  beforeEach(async () => {
+    jest.clearAllMocks();
+
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      // TODO remove AlertsModule and RecoveryModule once they're integrated in the AppModule
+      imports: [AppModule, AlertsModule, RecoveryModule],
```

Comment:
The `AlertsModule` doesn't need to be included as we are not testing the alerts routes.

Reply:
I removed it in 193e5cc012a581c84310250f9ebb28e0567a3ced.